/*
 * This file was automatically generated by EvoSuite
 * Sat May 24 19:54:40 GMT 2025
 */

package original;

import org.junit.Test;
import static org.junit.Assert.*;
import static org.evosuite.runtime.EvoAssertions.*;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;
import original.Solution2605;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true) 
public class Solution2605_ESTest extends Solution2605_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      Solution2605 solution2605_0 = new Solution2605();
      int[] intArray0 = new int[0];
      int int0 = solution2605_0.minNumber(intArray0, intArray0);
      assertEquals(100, int0);
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      Solution2605 solution2605_0 = new Solution2605();
      int[] intArray0 = new int[6];
      intArray0[2] = (-796);
      int int0 = solution2605_0.minNumber(intArray0, intArray0);
      assertEquals((-7960), int0);
  }

  @Test(timeout = 4000)
  public void test2()  throws Throwable  {
      int[] intArray0 = new int[71];
      Solution2605 solution2605_0 = new Solution2605();
      solution2605_0.minNumber(intArray0, intArray0);
      // Undeclared exception!
      solution2605_0.minNumber(intArray0, intArray0);
  }

  @Test(timeout = 4000)
  public void test3()  throws Throwable  {
      Solution2605 solution2605_0 = new Solution2605();
      int[] intArray0 = new int[9];
      // Undeclared exception!
      try { 
        solution2605_0.minNumber(intArray0, (int[]) null);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("original.Solution2605", e);
      }
  }
}
